vertica/datadog_checks/vertica/data/conf.yaml.example

```
@@ -148,6 +148,20 @@ instances:
     #     tags:
     #       - test:vertica
 
+    ## @param client_lib_log_level - string - optional
+    ## Vertica library log level.
+    ## If not defined, the vertica library logging is disabled.
+    ## Valid values: CRITICAL, FATAL, ERROR, WARN, WARNING, INFO, DEBUG
+    #
+    # client_lib_log_level: null
+
+    ## @param client_lib_log_path - string - optional
+    ## Vertica library log path.
+    ## This config has no effect if `client_lib_log_level` is not set.
+    ## Default to `/var/log/datadog/vertica_client_lib.log`
```

```suggestion
    ## Defaults to `/var/log/datadog/vertica_client_lib.log`
```
Small change, otherwise 👍

---

This section does not exist anymore.